modules/sticky-table-headers/main/index.coffee

```diff
@@ -258,28 +274,33 @@ class StickyTableHeaders
 
       topLeftTable = topLeftHead.append tableClone.clone(true)
 
-      realNodes = table.select('thead').selectAll('tr').select('th, td').nodes
-      clonedNodes = topLeftTable.select('thead').selectAll('tr').select('th, td').nodes
-
-      for i in [0...realNodes.length]
-        cloneEvents(realNodes[i], clonedNodes[i])
-        hx.select(realNodes[i]).classed('hx-sticky-table-invisible', true)
+      realNodes = getChildrenFromTable table, false, true
+      clonedNodes = getChildrenFromTable topLeftTable, false, true
 
+      createStickyHeaderNodes realNodes, clonedNodes
 
     topHead?.style('height', offsetHeight + 'px')
       .style('width', wrapperBox.width + 'px')
       .style('left', offsetWidth + 'px')
+      .selectAll('.hx-sticky-table-invisible')
```

Comment:
not quite sure what this is doing, but will this selection be okay if there are nested tables?